eraser-map.yaml

```diff
@@ -1466,7 +1462,7 @@ layers:
                         width: [[9, 0px], [10, 0.5px], [14, 2px], [16, 4m], [17, 5m], [18, 6m]]
                         outline:
                             color: [[9, [0.827,0.827,0.827]], [13, [0.827,0.827,0.827]], [15, [1.000,0.739,0.710]]]
-                            width: [[9, 0px], [10, 1px], [14, 1px], [16, 1.5px], [17, 2px]]
+                            width: [[13, 1px], [14, 1px], [16, 1.5px], [17, 2px]]
```

Comment:
(These are like text halos for roads, they add to the visual hierarchy in subtle ways that seem minor in isolation, but in aggregate across the whole screen make a difference.)